applications/LinearSolversApplication/custom_python/add_custom_decompositions_to_python.cpp

```diff
@@ -32,34 +32,34 @@ void AddCustomDecompositionsToPython(pybind11::module& m)
 {
     namespace py = pybind11;
 
-    typedef UblasSpace<double, Matrix, Vector> DenseSpaceType;
-    typedef typename DenseSpaceType::MatrixType MatrixType;
-    typedef typename DenseSpaceType::VectorType VectorType;
-    typedef DenseSingularValueDecomposition<DenseSpaceType> BaseSVDType;
+    // typedef UblasSpace<double, Matrix, Vector> DenseSpaceType;
```

Comment:
I meant to comment everything in this file, most importantly the includes!

Reply:
I forgot the includes :facepalm: my bad